pkg/organization/repository.go

```diff
@@ -46,17 +48,17 @@ func (r *sqlRepo) GetConfig(orgID string) (*Config, error) {
 	return &cfg, nil
 }
 
-func (r *sqlRepo) UpdateConfig(orgID string, companyID string) (bool, error) {
+func (r *sqlRepo) UpdateConfig(orgID string, cfg *client.OrganizationConfiguration) (*client.OrganizationConfiguration, error) {
 	query := `update organization_configs set company_identification = ? where organization = ? limit 1;`
```

Comment:
Did you intend to have a `limit` on this `update`? Apparently you have to set `SQLITE_ENABLE_UPDATE_DELETE_LIMIT` in SQLite if you want to use a `limit` with an `update`.

Reply:
Also, maybe I'm missing something, but is there a way to `insert` a _new_ configuration? This will only work if a config already exists.

Reply:
I didn't. Good catch.

Reply:
Sorry... Did you see my other question? I was wanting to test this out, but the only way to test it is to manually insert a row in `organization_configs`. Is there an endpoint that handles that?

Reply:
> Also, maybe I'm missing something, but is there a way to `insert` a _new_ configuration? This will only work if a config already exists.

my bad on that. the query needed to be a `replace into`. pushed